### DISCUSSION TITLE
Possible duplication of images in _site folder

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#921] [Get Help]
We use a Netlify continuous deploy for both of our projects, so I usually don't look too closely to the _site folder. We're thinking about hosting our projects on our own servers/website, though, so I zipped up the latest version of the _site folder today, and noticed that the images appear in two spots. All of the images in our Italian Paintings project (around 400 jpegs, totaling around 400 Mb) appear in both the main _assets folder, and in _assets>images in the two folders I made for cats and figs. Is this correct? It seems odd that the images would be saved in two places. This makes our _site folder around 800 Mb, and it would be great if it could instead be half the size. Thank you!

**Comment by Erin-Cecele** [comment]
Hi @zsofiaj, if your images are stored in `content/_assets/images/cats` and  `content/_assets/images/figs` then those are the only places they should exist in your `_site` folder. The only exception might be if you have iiif images. Then they would appear in `_site/iiif`. Can you send the Terminal output for when you run `quire build`? That might give us an indication as to why things are duplicating.

**Reply by zsofiaj** [reply]
Thanks so much for checking it out! Here is the Terminal output. Yes, images are stored in those two folders. We do have IIIF images, but they are all external/preexisting IIIF images. There is no iiif folder in my _site folder. 

[Zsofia_Terminal_output_quire_build.txt](https://github.com/thegetty/quire/files/15011142/Zsofia_Terminal_output_quire_build.txt)

**Reply by Erin-Cecele** [reply]
That's very interesting. I see in the Terminal output that the images are being "built" in the `_assets` folder. However, I just tried building the project using the `master` branch and did not have an issue with images being duplicated. Can you delete the `_site` folder and the `public` folder and rerun `quire build`.

**Reply by zsofiaj** [reply]
Thank you for taking a look @Erin-Cecele ! I deleted the _site folder and the public folder, and reran quire build, and the same thing happens. The images are built in both the _assets folder, and in the figures and cats folders where they are supposed to be.

**Reply by Erin-Cecele** [reply]
Hi @zsofiaj, I'm still having trouble recreating this issue on my end. Can you please run `quire info --debug` and share the output with me?

**Reply by zsofiaj** [reply]
Thank you, @Erin-Cecele ! Here's what I get when I run quire info --debug:

Project:
quire-cli 1.0.0-rc.11
quire-11ty 1.0.0-rc.16
starter https://github.com/thegetty/quire-starter-default@2.9.0

System:
quire-cli 1.0.0-rc.11
node v20.4.0
npm 9.7.2
os Darwin 23.3.0

**Comment by Erin-Cecele** [comment]
Hi @zsofiaj, thanks for sending that info. I'm having trouble recreating this issue on my end. I've been using the `master` branch. Is that correct?

**Reply by zsofiaj** [reply]
Yes, @Erin-Cecele, I'm just using the master branch as well. So odd that it doesn't do the same for you. Just as an experiment, I tried deleting the images from the _assets folder, leaving them only in the figs and cats folders, then compressing the _site folder and using Netlify Drop to deploy the site. It didn't work without the images in the main _assets folder. All of the images were missing.

**Comment by Erin-Cecele** [comment]
@zsofiaj I was finally able to recreate this issue! I downloaded a different node version, deleted the `node_modules` folder, ran `npm install`, deleted the `public`, `_site`, and `_epub` folders (also achieved by running `quire clean`) and then I ran `quire build` and saw the images created in both _assets and `_assets/images/figures` and `_assets/images/cats`. Still not sure why this is happening though!

**Comment by zsofiaj** [comment]
Glad you were able to recreate it @Erin-Cecele ! It is very odd. Even odder that when I change the base url to the Gallery's server address, the images in the _assets folder disappear.

**Comment by lucafalefoto** [comment]
I have the same problem as zsofiaj.

**Reply by Erin-Cecele** [reply]
Hi @lucafalefoto. Thanks for letting us know you are experiencing the same issue. Please run the command `quire info --debug` and send me the relevant terminal output. Thank you!

**Reply by lucafalefoto** [reply]
Here it is, Thank you.

> CLI] Command 'info' called with options { debug: true }
> This project was generated with the quire-cli prior to version 1.0.0.rc-8. Updating the version file to the new format, though this project's version file will not contain specific starter version information.
> [padova-decorata]
>  quire-cli <=1.0.0.rc-7
>  quire-11ty 1.0.0-rc.6
>  starter undefined
> [System]
>  quire-cli 1.0.0-rc.11
>  node v18.13.0
>  npm 9.2.0
>  os Linux 6.2.0-39-generic
